recipes/nosexcover/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{%set version = "1.0.10" %}
+package:
+  name: nosexcover
+  version: {{ version }}
+
+source:
+  fn: nosexcover-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/n/nosexcover/nosexcover-{{ version }}.tar.gz
+  md5: 12bf494a801b376debeb6a167c247391
+
+build:
+  preserve_egg_dir: True
+
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - nose
+    - coverage >=3.4
+
+test:
+  imports:
+    - nosexcover
+
+about:
+  home: http://github.com/cmheisel/nose-xcover/
+  license: BSD 1-Clause
```

Comment:
@pmlandwehr I believe the actual license is MIT. See https://github.com/cmheisel/nose-xcover/issues/10

The `setup.py` metadata is wrong BTW. (And I don't believe there is a BSD 1-Clause :smile:)